### PR TITLE
Add quiet parameter to KubeCluster

### PR DESCRIPTION
**Pull request by droctothorpe**
This parameter is available in the parent class but not exposed in `KubeCluster`.

### CODE REVIEW

File: dask_kubernetes/operator/kubecluster/kubecluster.py [hunk 1]
```diff
@@ -224,6 +228,7 @@ def __init__(
         self.idle_timeout = dask.config.get(
             "kubernetes.idle-timeout", override_with=idle_timeout
         )
+        self.quiet = quiet
```
Comment (jacobtomlinson) [inline dask_kubernetes/operator/kubecluster/kubecluster.py:231]: I think we should remove this and instead pass `quiet` along to the `super().__init__(..., quiet=quiet)` further down.

File: dask_kubernetes/operator/kubecluster/kubecluster.py [hunk 2]
```diff
@@ -115,6 +115,9 @@ class KubeCluster(Cluster):
         used to create the cluster instead of generating one from the other keyword arguments.
     scheduler_forward_port: int (optional)
         The port to use when forwarding the scheduler dashboard. Will utilize a random port by default
+    quiet: bool
+        If enabled, suppress rich console output.
```
Comment (jacobtomlinson) [inline dask_kubernetes/operator/kubecluster/kubecluster.py:119]: I think we should be clear that this suppresses all printed output from the cluster, not just the rich console. I acknowledge that the only output we have today is the rich console but that may not always be true.

File: dask_kubernetes/operator/kubecluster/kubecluster.py (hunk 1 above)
Comment (droctothorpe) [reply dask_kubernetes/operator/kubecluster/kubecluster.py]: Duh, heh. Not sure why I didn't think of that. Done.

**Review by jacobtomlinson — APPROVED** [review]
Awesome thanks!

**Comment by droctothorpe** [comment]
Parsing the test failures. I don't think they're related to this PR.

**Comment by jacobtomlinson** [comment]
Sorry for the delay in unblocking the CI here. Things should be good to go now.

**Comment by droctothorpe** [comment]
Thanks, @jacobtomlinson!

**Pull request merged**